### PR TITLE
Changes on the GUI of the game 

### PR DESCRIPTION
**Pull request by Dencruz**
Changes:
-Added some instructions to the empty space beside the grid for manually placing the ships.
-Enabled the close button of the window where the user will manually place their ships
-Fixed the issue with the invalid placement of ships

### CODE REVIEW

File: game.py [hunk 1]
```diff
@@ -1,41 +1,487 @@
...
+AI_grid = [[0] * GRID_SIZE for _ in range(GRID_SIZE)]
+
+# Array for AI's used attacks
+AI_shots = []
+AI_hits = []
+
+
+# Hit flag
+hitflag = 0
+
```
Comment (YouDontSayIt-dev) [inline game.py]: Why did the separated functions came back? Dapat nakahiwalay na sya

**Review by YouDontSayIt-dev — CHANGES_REQUESTED** [review]
Please address the changes requested

**Review by Kenshin0224 — COMMENTED** [review]
oks

File: game.py (hunk 1 above)
Comment (Dencruz) [reply game.py]: nagcommit na meee nung maayos naaa. maya ko na ulit tuloy HAHAHAHA

**Pull request closed without merging**